capstone/src/main/java/com/google/univiz/api/VisResource.java

```diff
@@ -1,8 +1,8 @@
 package com.google.univiz.api;
```

Comment:
Note that you're moving this class in #77. Make sure it ends up in the right place after resolving the merge conflict.